codes.md

```diff
@@ -125,19 +145,115 @@ Codes
+    - "I was thinking of a derivative of this dataset -- basically, after you perform named entity recognition, the derivative dataset might be conceptualized of as a network." (from Initial Data Abstraction)
+    - “One thing is it seems not easy to describe these things easily. I also sometimes feel should I talk about my raw data or formatted data. Although both are in tabular format. I also consider about wide-narrow tabular structure as well, probably it does not make sense, but sort of thinking about that somehow.” (from Initial Dataset Debrief)
+  - metadata as data
+    - "It’s tabular insofar as I’m tagging each observation with dates and other metadata; it’s useful to sort on that attribute" (from Initial Tabular Data Details)
+    - “Metadata is a large component of earth science data but I'm uncertain if I should have included it into this survey. I kept things simple, but may have perhaps been too simple.” (from Initial Dataset Debrief)
+    - “I think it could be done, a 3D volume is in some sense just a set of stacked 2D tables, but I'm not sure that is what this question is really getting at. I'm not sure if it would make sense to flatten volume data into a tabular form, I could believe that there are some derived quantities or meta data that would be useful to view this way, but not sure if transforming raw volume data would be that useful. I think that is a larger interesting question about this project -- to what extent does it connect with volume data or 3D simulation data, imaging, the sort of things that are common in sci vis?” (from Alternative Dataset Reflections)
+  - links to data in the data
+    - “There are links to media in the dataset, but the media themselves are not directly stored in the dataset.” (from Initial Media Data Details)
+  -  “the data in this way is more complicated to describe, but in the real world is more offen.” (from Alternative Dataset Reflections)
+  -  “Oftentimes, for certain types of time-series meteorological data is collected from one sensor that logs three values at one time stamp: min, max, avg. This does provoke an idea that data at certain time steps are related and can be related in a "tree like structure" but it also causes problems in visualizing the data because metadata is encoded along with data. This may muddle the purity of visualizing the data by itself.” (from Alternative Dataset Reflections) -- note the use of the word "muddle"
+  - Counts of how many people listed their data under multiple forms
+    - Media was the largest alternative category --- was it because it was the
+      only choice remaining for some participants?
+
+## LATENT_WRANGLING_OR_ABSTRACTION. This may be related to "HARD_TO_ISOLATE". Even before the alternative abstractions, people were discussing how they could see data in other forms.
```

Comment:
This is probably a better way to phrase the AUTOMATIC_DERIVATION_INFERENCE code; any objections if I put its evidence under this header?

Reply:
(I can do that in a later PR)

Reply:
Yes, definitely move things around/re-word as they make sense.